scio-parquet/src/main/scala/com/spotify/scio/parquet/tensorflow/package.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.parquet
+
+import com.spotify.scio.ScioContext
+import com.spotify.scio.io.ClosedTap
+import com.spotify.scio.parquet.tensorflow.ParquetExampleIO.WriteParam
+import com.spotify.scio.values.SCollection
+import me.lyh.parquet.tensorflow.Schema
+import org.apache.parquet.filter2.predicate.FilterPredicate
+import org.apache.parquet.hadoop.metadata.CompressionCodecName
+import org.tensorflow.example.Example
+
+/**
+ * Main package for Parquet TensorFlow APIs. Import all.
+ *
+ * {{{
+ * import com.spotify.scio.parquet.tensorflow._
+ * }}}
+ */
+package object tensorflow {
+
+  /** Enhanced version of [[ScioContext]] with Parquet [[Example]] methods. */
+  implicit class ParquetExampleScioContext(private val self: ScioContext) extends AnyVal {
```

Comment:
could we follow the same syntax structure as we have in the other modules?

Reply:
That'd "require" redoing `parquet.avro` for consistency's sake? What's the add-on value? Other than binary breakage?

Reply:
yeah the main value is consistency and that we should avoid having this in package objects. We can do it in another PR.